<ui version="4.0" >
 <class>BackendCapabilitiesWidget</class>
 <widget class="QWidget" name="BackendCapabilitiesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>448</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Available Audio Devices:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Supported MIME Types:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QListView" name="audioDevicesListView" />
   </item>
   <item row="1" column="1" >
    <widget class="QListWidget" name="mimeTypesListWidget" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Available Audio Effects:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QTreeWidget" name="audioEffectsTreeWidget" >
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
